cmd/openshift-sdn/openshift-sdn.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"math/rand"
+	"os"
+	"time"
+
+	"k8s.io/apiserver/pkg/util/logs"
+
+	"github.com/openshift/origin/pkg/cmd/flagtypes"
```

Comment:
@juanvallejo you're familiar with cobra bits. Can you see if this is a thing we need to move to library-go?  We need to see who is using it and what for.  We may just be able to kill all other consumers

Reply:
That package is used by the following files in Origin:

```
pkg/oc/cli/login/login.go
pkg/oc/cli/cli.go
pkg/cmd/server/start/start_controllers.go
pkg/cmd/server/start/master_args.go
pkg/cmd/server/start/start_api.go
pkg/cmd/server/start/kube_connection_args.go
pkg/cmd/server/start/listen_arg.go
pkg/cmd/server/admin/create_nodeconfig.go
pkg/cmd/server/kubernetes/master/master_config.go
pkg/cmd/openshift/openshift.go
```

The package allows us to bind entire structs to cobra flags, and handle how flag values are set into fields in that struct. Not sure that cobra has a built-in solution for this, other than providing the `Value` interface.
We also use it in `openshift.go` and `cli.go` to bind `glog` flags to a given flagset.

Reply:
@juanvallejo remember to come back to it once we kill openshift start